sherlock/internal/api/sherlock/git_commits_v3_upsert.go

```diff
@@ -8,15 +8,14 @@ import (
 	"github.com/broadinstitute/sherlock/sherlock/internal/models"
 	"github.com/gin-gonic/gin"
 	"net/http"
-	"time"
 )
 
 type GitCommitV3Upsert struct {
-	GitRepo      string    `json:"gitRepo"`
-	GitCommit    string    `json:"gitCommit"`
-	GitBranch    string    `json:"gitBranch"`
-	IsMainBranch bool      `json:"isMainBranch"`
-	CommittedAt  time.Time `json:"committedAt"`
+	CommonFields
```

Comment:
```suggestion
```
**suggestion:** I don't think we actually want to allow upserting the CommonFields, those are typically ready only (the underlying `gorm.Model` is handled by Gorm, so I actually don't know what would happen if you tried to manually set these)